docs/topics/webhooks/lambda-handler.md

```diff
@@ -0,0 +1,116 @@
+-   **Parameters**
+    -   _EventBridgeBus_: If using a custom bus, enter the name here. By default, the `default`` bus is used.
+    -   _MomentoSecretString_: The secret string that Momento uses to sign the payload and sends as a part of the HTTP Headers in the webhook POST. This secret string is available in the Momento Webhook Console. Copy from the below screen and paste it into this parameter
+
+![Momento Webhook Secret](./webhook_settings.jpg)
+
+### Update the Momento webhook URL
+
+Upon completion of the deployment, take the Function URL from the CloudFormation output and update the Momento webhook URL in the Momento console like the image one section above. Below is what the CloudFormation Output will look like:
+
+![CloudFormation Output](./cloudformation_output.jpg)
+
+## Verifying the Payload
+
+The ultimate destination for this webhook handler is AWS EventBridge. When working with EventBridge, Rules are the configuration point that drives what is filtered and what targets are provided the message to act upon.
+
+The Momento Lambda webhook handler puts each event from the webhook on the Event Bus with the following structure.
+
+### Example Event
+
+Below is a sample payload representing an EventBridge message as triggered from a Momento webhook. Note that this is a template and specific values will differ in your implementation.
+
+```json
+{
+    "version": "0",
+    "id": "7c7ce805-51c3-4a66-b6e0-39e4e558e6d8",
+    "detail-type": "sample-a#sample-a",
+    "source": "Momento",
+    "account": "252703795646",
+    "time": "2024-01-13T14:56:16Z",
+    "region": "us-west-2",
+    "resources": [],
+    "detail": {
+        "cache": "sample-a",
+        "topic": "sample-a",
+        "event_timestamp": 1705157775308,
+        "publish_timestamp": 1705157775308,
+        "topic_sequence_number": 2,
+        "token_id": "",
+        "text": "{\"someKey\":\"someValue\"}"
+    }
+}
+```
+
+### Event Structure
+
+The properties of the EventBridge message are listed below along with the function they provide.
+
+#### detail-type
+
+detail-type is used to help filter out messages based on the cache and topic source. Since the webhook handler can be used as a single gateway for multiple webhooks, filtering in EventBridge to drive certain values to certain targets is useful. This value will always be a concatenated string in the format of `{cache-name}#{topic-name}` which is sourced from the webhook payload. Note the `#` in the middle of the dynamic values.
+
+#### source
+
+The source will always be `Momento`. This property helps determine the origination point of the message.
+
+#### detail
+
+The message detail will contain the actual payload as supplied from the webhook. By not removing elements from the message, maximum detail is provided to all consuming applications. For reference, the Momento webhook payload fields are [described here.](https://docs.momentohq.com/topics/webhooks/overview#example-event)
+
+## Notes and Wrapping Up
```

Comment:
To make it feel less bloggy/story, Maybe change this to "Considerations" and remove the storytelling part of the first paragraph in here.